packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -358,8 +362,10 @@ export default class SalesforceCRMService implements CRM {
         )}')`
       );
 
-      if (contactSearch && contactSearch.records.length > 0)
+      if (contactSearch && contactSearch.records.length > 0) {
         records = contactSearch.records as ContactRecord[];
+        this.setFallbackToContact(true);
```

Comment:
If we're falling back to a contact, let's store that as a variable so in other methods, we know to query the contact objects.